doc_user/UserManual.adoc

```diff
@@ -1873,23 +1875,23 @@ Once the data is entered the *Buttons panel* has the following choices:
 
```

Comment:
There is a spelling error at line 1866:  `lenght` -> `length`

Reply:
Done, thank you !